In-house patch to remove unnecessary dependencies from Keystone's
requirements files. The specific reasons are as follows:

Paste			Not applicable

pysaml2			Not applicable to Solaris

--- keystone-9.0.2/requirements.txt.~1~	2016-05-26 11:34:30.000000000 -0800
+++ keystone-9.0.2/requirements.txt	2016-06-27 18:17:38.084276305 -0800
@@ -7,7 +7,6 @@ WebOb>=1.2.3 # MIT
 eventlet!=0.18.3,>=0.18.2 # MIT
 greenlet>=0.3.2 # MIT
 PasteDeploy>=1.5.0 # MIT
-Paste # MIT
 Routes!=2.0,!=2.1,!=2.3.0,>=1.12.3;python_version=='2.7' # MIT
 Routes!=2.0,!=2.3.0,>=1.12.3;python_version!='2.7' # MIT
 cryptography!=1.3.0,>=1.0 # BSD/Apache-2.0
@@ -32,7 +31,6 @@ oslo.serialization>=1.10.0 # Apache-2.0
 oslo.service>=1.0.0 # Apache-2.0
 oslo.utils>=3.5.0 # Apache-2.0
 oauthlib>=0.6 # BSD
-pysaml2<4.0.3,>=2.4.0 # Apache-2.0
 dogpile.cache>=0.5.7 # BSD
 jsonschema!=2.5.0,<3.0.0,>=2.0.0 # MIT
 pycadf!=2.0.0,>=1.1.0 # Apache-2.0
